### PR TITLE
feat: LT-1021 Licensed Quantity & Licence page

### PR DESCRIPTION
**Pull request by BenAndrew-Kainos**

### CODE REVIEW

File: cases/views/advice.py [hunk 1]
```diff
@@ -311,52 +312,61 @@ class Finalise(TemplateView):
     Finalise a case and change the case status to finalised
     """
 
+    @staticmethod
+    def _get_goods(request, pk, case_type):
+        goods = []
+        if case_type == CaseType.STANDARD.value:
+            goods, status_code = get_finalise_application_goods(request, pk)
+            if status_code != HTTPStatus.OK:
+                return error_page(request, "Approved goods could not be fetched")
```
Comment (andytaylorkainos) [inline cases/views/advice.py:321]: might be worth getting an Anthony-approved error message

File: cases/views/advice.py [hunk 2]
```diff
@@ -372,12 +382,14 @@ def post(self, request, *args, **kwargs):
             return error_page(request, "You do not have permission.")
 
         if res.status_code == 400:
```
Comment (andytaylorkainos) [inline cases/views/advice.py:384]: i know its not your code but can you update this to the status library please?

File: cases/views/advice.py (hunk 1 above)
Comment (BenAndrew-Kainos) [reply cases/views/advice.py]: will do

File: cases/views/advice.py (hunk 2 above)
Comment (BenAndrew-Kainos) [reply cases/views/advice.py]: done

**Pull request merged**